### ISSUE TITLE
What IP should you use to use solar stats? It says that my proxy is open, but whenever i try to connect using (Hypixel.net) (stats-proxy.hypixel.net) (localhost) none of them work and I need help.

### ISSUE THREAD

**Issue by jp3iii** [Solar-Tweaks/SolarStats#20]

**Comment by RatchanonDev** [comment]
Put this in your hosts file. `127.0.0.1 solarstats.hypixel.net` and connect using solarstats.hypixel.net:25556

**Comment by Husk-ees** [comment]
Yeah im having the same problem... Where would you put the "127.0.0.1 solarstats.hypixel.net"???

**Comment by RatchanonDev** [comment]
> Yeah im having the same problem... Where would you put the "127.0.0.1 solarstats.hypixel.net"???

In c:\windows\system32\drivers\etc\hosts

**Comment by Husk-ees** [comment]
Oh... I use mac

**Comment by Husk-ees** [comment]
NVM figured it out

**Comment by Husk-ees** [comment]
Error code comes up: "Internal exception: java.net.SocketException: Connection Reset" joining with solarstats.hypixel.net:25556 and yes i did change the host and ran the solar stats thingy

**Comment by Husk-ees** [comment]
OK again so I set the ip to solarstats.hypixel.net:25565 instead and it still doesn't work rip

**Comment by Husk-ees** [comment]
Wait nope ignore everything i said above it started to work and it asked me to sign in through the browser... and then... 
"
node:internal/process/promises:279
            triggerUncaughtException(err, true /* fromPromise */);
            ^

Error: getaddrinfo ENOTFOUND 127.0.0.1 solarstats.hypixel.net
    at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:71:26) {
  errno: -3008,
  code: 'ENOTFOUND',
  syscall: 'getaddrinfo',
  hostname: '127.0.0.1 solarstats.hypixel.net'
}
"
Help pls

**Comment by RatchanonDev** [comment]
I don't use mac so I don't really know.

**Comment by Husk-ees** [comment]
<img width="651" alt="Screen Shot 2022-07-19 at 8 34 29 AM" src="https://user-images.githubusercontent.com/62223780/179790545-be24417e-f126-4284-9b97-4a104bdbda94.png">
I GOT IT WORKING YOOO
